### PR TITLE
Modify Japanese translation

### PR DESCRIPTION
**Pull request by nabbisen**
## Description
These commits modify Japanese translation on #774 .

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I ran `bin/console kimai:codestyle --fix` to verify the correct code style
- [ ] I have updated the [documentation](https://github.com/kimai/www.kimai.org/tree/master/_documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
Could you have a look at these 3 new keys as well:
```
 $ bin/console debug:translation --only-missing ja
 ----------- ---------- --------------- ---------------------- ------------------------------------------
  State       Domain     Id              Message Preview (ja)   Fallback Message Preview (en)
 ----------- ---------- --------------- ---------------------- ------------------------------------------
   missing    messages   label.tags      label.tags             Tags
   missing    tags       tags.title      tags.title             Tags
   missing    tags       tags.subtitle   tags.subtitle          Tags allow, to freely combine timeshe...
 ----------- ---------- --------------- ---------------------- ------------------------------------------
```
Its for the tagging feature, which is new since yesterday.

**Comment by nabbisen** [comment]
Sure!
It seems `label.tag` / `label.tags` as well as `menu.tags` were added in messages.xx.xliff from my version.
I'll add all.

File: translations/tags.ja.xliff [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file date="2019-05-11T10:00:00Z" source-language="en" target-language="en" datatype="plaintext" original="none">
```
Comment (kevinpapst) [inline translations/tags.ja.xliff]: Can you please change this:
```
<file date="2019-05-11T10:00:00Z" source-language="en" target-language="ja" datatype="plaintext" original="tags.en.xliff">
```

Comment (nabbisen) [reply translations/tags.ja.xliff]: Oh, new file's header...
Sure!

**Comment by kevinpapst** [comment]
Thanks @nabbisen 😄

**Pull request merged**